tests/test_cases/test_discovery/test_discovery.py

```diff
@@ -476,16 +476,10 @@ async def access_block_vhdl(dut):
     """Access a VHDL block statement"""
 
     dut._log.info(
-        "Block: {} ({})".format(
-            dut.isample_module1.SAMPLE_BLOCK._name,
-            type(dut.isample_module1.SAMPLE_BLOCK),
-        )
+        f"Block: {dut.isample_module1.SAMPLE_BLOCK._name} ({type(dut.isample_module1.SAMPLE_BLOCK)})"
```

Comment:
I don't like these, but they really should use `%s` because they are log calls. No need to evaluate a string unless we need to.